builtin/json.mbt

```diff
@@ -47,8 +47,42 @@ pub fn Double::to_json(self : Double) -> Json {
   Number(self)
 }
 
+fn escape_json_string(str : String) -> String {
+  fn to_hex_digit(i : Int) -> Char {
+    if i < 10 {
+      Char::from_int('0'.to_int() + i)
+    } else {
+      Char::from_int('a'.to_int() + (i - 10))
+    }
+  }
+
+  let buf = Buffer::new(size_hint=str.length())
+  for i = 0; i < str.length(); i = i + 1 {
+    let c = str[i]
+    match c {
+      '\n' => buf.write_string("\\n")
```

Comment:
Theres no need to escape `"` `\` here because it can be captured by the language